amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-studio.adoc

```diff
@@ -1,94 +1,177 @@
+
 +
-. If you are:
+You can reference a configuration file that contains ANT-style property placeholders (recommended), or you can enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
 +
-. If you are using the default Amazon S3 storage, leave the default entries for *Proxy*, *Security*, and *Advanced* tabs.
-. If you are connecting to different storage than the default AWS S3, specify its URL in the *Advanced* tab's *S3 Compatible Storage URL* field.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
 
-. Click *Test Connection* to confirm that the parameters of your global configuration are accurate and that Mule is able to successfully connect to your instance of Amazon S3.
 +
 Validating a connection with *Test Connection* requires that you have permission in AWS IAM to the action `s3:ListAllMyBuckets`. If you don’t have this permission, the test fails. However, you can still use the connector and the operations to which you have access.
 +
 Access to operations on Amazon S3 is further controlled through policies. It is not always possible to validate your credentials before the exact operation for which you have access completes. This can vary based on the bucket name and other parameters. For example, the test connection can fail if your credentials have a restricted policy.
 
-. Click *OK* to save the global connector configurations.
+[[basic-authentication]]
+=== Basic Authentication
+
+. Enter the following information on the *General* tab of the *Global Element Properties* screen to configure Basic authentication:
+
+[%header,cols="30s,70a"]
+|===
+|Field |User Action
+|Name |Enter the configuration name.
+|Connection | Select `Basic`.
+|Session Token | Optionally enter the session token provided by Amazon Security Token Service (STS).
+|Access Key | Enter the access key provided by Amazon.
+|Secret Key | Enter the secret key provided by Amazon.
+|Region Endpoint | Select the region endpoint for the service.
+|===
+
+. On the *Advanced* tab, set the 'Try Default AWS Credentials Provider Chain' field to 'true' to obtain credentials from the AWS environment.
+
+The following screenshot shows an example of configuring the *General* tab for Basic authentication:
+
+.Basic authentication fields
+image::amazon-s3-01.png[To configure Basic authentication, select `Basic` in the *Connection* field and then complete the fields on the *General* tab.]
+
+[[role-authentication]]
+=== Role Authentication
+
+. Enter the following information on the *General* tab of the *Global Element Properties* screen to configure Role authentication:
```

Comment:
The formatting is not working here. I think you need to put a plus sign before and after the table.